### PR TITLE
Update Android Platform Support

### PR DESCRIPTION
**Pull request by christophaigner**
## Description
This PR updates the Android/Android TV/Fire TV Platform Support (as defined in Bitmovin's [Supported Platforms & Devices](https://developer.bitmovin.com/playback/docs/supported-platforms-devices-player)).

## Checklist
- [ ] 🗒 `CHANGELOG` entry

### CODE REVIEW

**Review by matamegger — APPROVED** [review]
Thanks for streamlining the versions and adding a reference 👍

File: README.md [hunk 1]
```diff
@@ -12,13 +12,12 @@ This is an open-source project created to enable customers to integrate the Bitm
...
 
-This library requires at least React Native 0.64+ and React 17+ to work properly. The currently supported platforms are:
+This library requires at least React Native 0.64+ and React 17+ to work properly. The **officially supported** platforms are:
 
-- iOS 14.0+
-- tvOS 14.0+
-- Android API Level 21+
-- Android TV API Level 24+
-- Fire TV FireOS 5.0+
+- **iOS/tvOS:** 14.0+
```
Comment (christophaigner) [inline README.md:17]: @rolandkakonyi @123mpozzi what about iPadOS actually? Is that also supported?

Comment (christophaigner) [reply README.md:17]: Still merging this PR as this is for Android, but if iPadOS is also supported, please update the README again :)

Comment (rolandkakonyi) [reply README.md:17]: @christophaigner iPadOS is a marketing name only, we do nothing for or against to support it, but I will add it there.
Commit: https://github.com/bitmovin/bitmovin-player-react-native/commit/ac05c98ca9a0f0f718e541f8398d443eaf8b2220

**Pull request merged**